packages/nameguard-sdk/src/index.test.ts

```diff
@@ -139,7 +139,7 @@ describe("NameGuard", () => {
     );
 
     expect(data.primary_name_status).toBe("unnormalized");
-    expect(data.impersonation_status).toBeNull();
+    expect(data.impersonation_estimate).toBeNull();
```

Comment:
I see a "Check failure" on this line inside GitHub's PR review interface. Could you please investigate? Thanks